client/my-sites/stats/post-trends/index.jsx

```diff
@@ -152,7 +152,7 @@ class PostTrends extends React.Component {
 		return (
 			<div className="post-trends">
 				{ siteId && <QuerySiteStats siteId={ siteId } statType="statsStreak" query={ query } /> }
-				<SectionHeader label={ this.props.translate( 'Posting Activity' ) } />
+				<SectionHeader label={ this.props.translate( 'Posting activity' ) } />
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.